suse_migration_services/units/ssh_keys.py

```diff
@@ -47,6 +52,34 @@ def main():
         log.info('Save keys to {0}'.format(migration_ssh_file))
         with open(migration_ssh_file, 'w') as authorized_migration_file:
             authorized_migration_file.write(authorized_keys_content)
+
+        if os.path.exists(os.path.dirname(system_ssh_host_keys_glob_path)):
+            system_ssh_host_keys = glob.glob(system_ssh_host_keys_glob_path)
```

Comment:
It's not needed to check for the presence of the directory. The glob match will return an empty list for any reason of the pattern not matching. Thus it's enough to go with:

```python
system_ssh_host_keys = glob.glob(system_ssh_host_keys_glob_path)
if system_ssh_host_keys:
    ...
```